catalyst/utils/config.py

```diff
@@ -1,23 +1,24 @@
-from typing import Any, Dict, List, Union  # isort:skip
 from collections import OrderedDict
 import copy
+from itertools import chain
 import json
-from logging import getLogger
+import logging
 import os
 from pathlib import Path
 import platform
 import re
 import shutil
 import subprocess
 import sys
+from typing import Any, Dict, List, Union
```

Comment:
Unfortunately, there is a bug in `isort` so it is not able to sort standard python libs in the custom order. Could you move `from typing import Any, Dict, List, Union # isort:skip` line to the top of the file?

Reply:
fixed